app/models/trainee.rb

```diff
@@ -304,6 +304,8 @@ class Trainee < ApplicationRecord
   COMPLETE_STATES = %w[recommended_for_award withdrawn awarded].freeze
   IN_TRAINING_STATES = %w[submitted_for_trn trn_received recommended_for_award].freeze
 
+  attr_accessor :ethnicity
+
```

Comment:
Rather than adding an `attr_accessor` here, could we remove the ethnicity value from the attributes before we try to save the Trainee? So maybe change the [deep_attributes](https://github.com/DFE-Digital/register-trainee-teachers/blob/ae69877b1bcb43aac7f9b1e42fec37581fe25290/app/models/api/trainee_attributes/v01.rb#L158) method in TraineeAttributes, so that we're still validating the ethnicity, but we don't try to create a trainee with it?